de_build/build-adf-spark-job-new.yml

```diff
@@ -0,0 +1,73 @@
+parameters:
```

Comment:
do we need both `build-adf-spark-job-new.yml` and `build-adf-spark-job.yml`?

Reply:
build-adf-spark-job-new.yml is not required ,will be removing it.

Reply:
removed